CMakeLists.txt

```diff
@@ -163,7 +163,11 @@ CHECK_STRUCT_HAS_MEMBER("struct stat" st_mtim.tv_nsec sys/stat.h
 CHECK_STRUCT_HAS_MEMBER("struct stat" st_mtimespec.tv_nsec sys/stat.h
   HAVE_STAT_ST_MTIMESPEC_TV_NSEC LANGUAGE C)
 
-set(CEPH_MAN_DIR "share/man" CACHE STRING "Install location for man pages (relative to prefix).")
+if(FREEBSD)
+  set(CEPH_MAN_DIR "man" CACHE STRING "Install location for man pages (relative to prefix).")
```

Comment:
@tchaikov 
Reasons I did it this way are:
1) Don't really know if ever anybody is going to rumage thru the CACHE looking for the STRING.
2) The ports building does not use ./do_freeBSD.sh because the way external sources are accessed is not how it should be done in port.
But I can do this in the ports Makefile 
3) This is a rather permanent  fix for building on FreeBSD, and I'm using the stuff in the do_freebsd.sh script more for that can change in the future. And it fixes it for all build environments at once, I currently have: Ceph, jenkins, ports, pouderiere.

So I would really appreciate it to go here.